rke/k8s_service_options.go

```diff
@@ -192,3 +200,10 @@ func getSchedulerOptions() map[string]string {
 		"profiling":    "false",
 	}
 }
+
+func getETCDOptions() map[string]string {
```

Comment:
Shouldn't we add these etcd options to the other k8s versions > 1.13?

Reply:
could add, i thought cis scan ran only for 1.13